view/template/patient.hbs

```diff
@@ -1,3 +1,116 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
+<head>
+    <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.4.1/jquery.js"></script>
+    <script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.8.14/jquery-ui.min.js"></script>
+    <link rel="stylesheet" type="text/css" media="screen" href="http://ajax.googleapis.com/ajax/libs/jqueryui/1.8.14/themes/base/jquery-ui.css">
+<style>
+.highlight {
+    border: 1px solid red;
+}
+
+
+.highlight2 {
+    border: 1px solid green;
+}
+
+</style>
```

Comment:
`doctype`, `html`, `head`, and `script`s are inherited from `view/layout/default.hbs`
Add the new scripts there, and any other meta data you need set.